server/src/film/film.controller.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  IInfiniteScrollRequest,
+  IPaginationRequest,
+} from 'shared/types/pagination';
+import filmService from './film.service';
+import { Response } from 'express';
+
+async function getPage(req: IPaginationRequest, res: Response) {
+  const limit = Number(req.query.limit);
+  const pageNumber = Number(req.query.pageNumber);
+  const films = await filmService.paginate(limit, pageNumber);
+  return res.status(200).json({ films });
```

Comment:
Nitpick: is the status 200 necessary?

Reply:
Just a personal habit to always specify status codes, even when they are sent automatically.